sdk/eventhub/event-hubs/README.md

```diff
@@ -276,15 +276,21 @@ To control the number of events passed to processEvents, use the options argumen
 You can use `EventHubClient` to work with IotHub as well. This is useful for receiving telemetry data of IotHub from the linked EventHub.
 Most likely the associated connection string will not have send claims. Hence getting HubRuntimeInfo or PartitionRuntimeInfo and receiving events would be the possible operations.
 
-- Please notice that we are awaiting on the [createFromIotHubConnectionString](https://azure.github.io/azure-sdk-for-js/event-hubs/classes/eventhubclient.html#createfromiothubconnectionstring) method to get an instance of the EventHubClient. This is different from other static methods on the client. The method talks to the IotHub endpoint to get a redirect error which contains the EventHub endpoint to talk to. It then constructs the right EventHub connection string based on the information in the redirect error and returns an instance of the EventHubClient that you can play with.
+- Please notice that the connection string needs to contain the `Endpoint` key-value pair
+  in addition to the `SharedAccessKeyName` and `SharedAccessKey` key-value pairs.
```

Comment:
Should we instead talk directly about the "Built in endpoints" or "event hub compatible endpoints" instead?
If they have the iot hub specific endpoint, and they read this line, we are telling them that it wont work, but we are not telling them how to fix the issue and move ahead